<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="QDialog" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credit Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	 background-color: rgb(236, 240, 241);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="9" column="0">
    <widget class="QLabel" name="result_show_6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string> monthly payment</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="result_show_8">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string> debt + interest</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="credit_show">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
   color: black;
   font-size: 16px;
   background-color : white;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QPushButton" name="pushButton_calculate">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
  font-size: 16px;
  color: white;
  background-color: rgb(255, 157, 3);
}
QPushButton::pressed{
  background-color: rgb(244, 129, 2);
}</string>
     </property>
     <property name="text">
      <string>calculate</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="interest_sign">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string> %</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="1">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QComboBox" name="type_show">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
  color: black;
  font-size: 15px;
  background-color: white;
}
</string>
     </property>
     <item>
      <property name="text">
       <string>annually</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>differentiated</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="1" colspan="3">
    <widget class="QLabel" name="clean_int_show">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{   
  color: black;
   font-size: 16px;
   background-color : white;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QComboBox" name="period_term">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
  color: black;
  font-size: 15px;
  background-color: white;
}
</string>
     </property>
     <item>
      <property name="text">
       <string>years</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>months</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="result_show_7">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string> accrued interest</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="1" colspan="3">
    <widget class="QLabel" name="debt_plus_int_show">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{   
  color: black;
   font-size: 16px;
   background-color : white;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="result_show_3">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string> interest rate</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="result_show_2">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string> loan term</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="term_show">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
   color: black;
   font-size: 16px;
   background-color : white;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="result_show_4">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string> type of payments</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="3">
    <widget class="QLabel" name="payment_show">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{   
  color: black;
   font-size: 16px;
   background-color : white;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="interest_show">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
   color: black;
   font-size: 16px;
   background-color : white;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="result_show">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-size: 16px;
  color: black;
}

</string>
     </property>
     <property name="text">
      <string> amount of credit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
